src/ipc.c

```diff
@@ -122,15 +123,16 @@ umf_result_t umfPutIPCHandle(umf_ipc_handle_t umfIPCHandle) {
 umf_result_t umfOpenIPCHandle(umf_ipc_handler_handle_t hIPCHandler,
                               umf_ipc_handle_t umfIPCHandle, void **ptr) {
 
+    umf_memory_provider_handle_t hProvider = hIPCHandler;
+
     // IPC handler is an instance of tracking memory provider
```

Comment:
the comment here should be updated, I guess...? (and the one below?)

Reply:
is it?

Reply:
I don't understand what you mean. Could you please clearly specify what I should change and how?